experimental/dds/tree2/src/feature-libraries/sequence-field/format.ts

```diff
@@ -12,7 +12,24 @@ import { ChangesetLocalId, NodeChangeset } from "../modular-schema";
 export type ProtoNode = JsonableTree;
 
 export type NodeCount = number;
-export type Skip = number;
+
+/**
+ * Left undefined for terseness.
+ */
+export const SkipType = undefined;
+
+export interface Skip extends CellTargetingMark {
```

Comment:
While it's fine for there to be a skip mark targeting a detached cell, it is unnecessary for us to ever create them.

Reply:
I considered add these fields to capture that in the type system
```
	detachEvent?: undefined;
	lineage?: undefined;
```
but when we refactor to orthogonalize the where/anchor from the what/effect we'll likely tolerate such a thing anyway.